src/Renderer/JsonRenderer.php

```diff
@@ -19,20 +19,30 @@ final class JsonRenderer implements ThrowableRendererInterface
 {
     public function render(Throwable $t, ServerRequestInterface $request = null): ErrorData
     {
-        return new ErrorData(json_encode([
-            'message' => self::DEFAULT_ERROR_MESSAGE,
-        ], JSON_THROW_ON_ERROR | JSON_UNESCAPED_SLASHES));
+        return new ErrorData(
+            json_encode(
+                [
+                    'message' => self::DEFAULT_ERROR_MESSAGE,
+                ],
+                JSON_THROW_ON_ERROR | JSON_UNESCAPED_SLASHES
+            )
+        );
     }
 
     public function renderVerbose(Throwable $t, ServerRequestInterface $request = null): ErrorData
     {
-        return new ErrorData(json_encode([
-            'type' => get_class($t),
-            'message' => $t->getMessage(),
-            'code' => $t->getCode(),
-            'file' => $t->getFile(),
-            'line' => $t->getLine(),
-            'trace' => $t->getTrace(),
-        ], JSON_THROW_ON_ERROR | JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_INVALID_UTF8_SUBSTITUTE));
+        return new ErrorData(
+            json_encode(
+                [
+                    'type' => get_class($t),
+                    'message' => $t->getMessage(),
+                    'code' => $t->getCode(),
+                    'file' => $t->getFile(),
+                    'line' => $t->getLine(),
+                    'trace' => $t->getTrace(),
+                ],
+                JSON_THROW_ON_ERROR | JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_INVALID_UTF8_SUBSTITUTE | JSON_PARTIAL_OUTPUT_ON_ERROR
```

Comment:
```suggestion
                JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_INVALID_UTF8_SUBSTITUTE | JSON_PARTIAL_OUTPUT_ON_ERROR
```

[PHP docs](https://www.php.net/manual/en/json.constants.php):
> JSON_PARTIAL_OUTPUT_ON_ERROR takes precedence over JSON_THROW_ON_ERROR